independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/CustomAlterableContexts.java

```diff
@@ -0,0 +1,64 @@
+package io.quarkus.arc.processor;
+
+import java.lang.reflect.Modifier;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.function.Predicate;
+
+import jakarta.enterprise.context.spi.AlterableContext;
+import jakarta.enterprise.inject.spi.DeploymentException;
+
+import org.jboss.jandex.DotName;
+
+public class CustomAlterableContexts {
```

Comment:
Does it have to be public?

Reply:
It is used from the `io.quarkus.arc.processor.bcextensions` package.

Reply:
Ok then...